contracts/chain-adapters/ZkStack_Adapter.sol

```diff
@@ -21,8 +21,11 @@ import { BridgeHubInterface } from "../interfaces/ZkStackBridgeHub.sol";
 contract ZkStack_Adapter is AdapterInterface {
     using SafeERC20 for IERC20;
 
+    // The ZkSync bridgehub contract treats address(1) to represent ETH.
+    address private constant ETH_TOKEN_ADDRESS = address(1);
```

Comment:
Is this token an ERC20? that's so weird

Reply:
It looks like they just use it as a special case for ETH https://github.com/matter-labs/era-contracts/blob/aafee035db892689df3f7afe4b89fd6467a39313/l1-contracts/contracts/bridge/L1SharedBridge.sol#L326